Index: JVoiceXmlImplementationPlatform.java
===================================================================
--- JVoiceXmlImplementationPlatform.java	（版本 1233）
+++ JVoiceXmlImplementationPlatform.java	（工作副本）
@@ -376,6 +376,15 @@
         if (LOGGER.isDebugEnabled()) {
             LOGGER.debug("waiting for empty output queue...");
         }
+        
+        // add by zhang nan, begin
+        try {
+            Thread.sleep(1000);
+        } catch (InterruptedException e1) {
+            // TODO Auto-generated catch block
+            e1.printStackTrace();
+        }
+        // end 
 
         if (LOGGER.isDebugEnabled()) {
             LOGGER.debug("...output queue empty.");
